src/style.css

```diff
@@ -1,3 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --main-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  background: #FFF;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo-link {
+  margin-right: auto;
```

Comment:
that it is not the best implementation, lets do `justify-content: space-between` for header